### PR TITLE
Create interface for security Part1

### PR DESCRIPTION
**Pull request by yyongycy**
### What changes are proposed in this pull request?

Create interface for security

### Why are the changes needed?
Create interface for security

### Does this PR introduce any user facing changes?
NA

### CODE REVIEW

File: dora/core/common/src/main/java/alluxio/wire/WorkerNetAddress.java [hunk 1]
```diff
@@ -31,6 +31,7 @@
 @NotThreadSafe
 public final class WorkerNetAddress implements Serializable {
   private static final long serialVersionUID = 0L;
+  private int mSecureRpcPort;
```
Comment (dbw9580) [inline dora/core/common/src/main/java/alluxio/wire/WorkerNetAddress.java:34]: put this next to `mRpcPort` and `mDataPort`

**Review by dbw9580 — APPROVED** [review]
LGTM

**Comment by yyongycy** [comment]
alluxio-bot, merge this please

**Comment by yyongycy** [comment]
alluxio-bot, merge this please

**Pull request merged**